apps/jubmoji-quest/src/components/LeaderBoard.tsx

```diff
@@ -37,37 +68,45 @@ export default function LeaderBoard() {
           <RankLabel variant="primary">Name</RankLabel>
           <RankLabel variant="primary">Score</RankLabel>
         </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel variant="active">Rank</RankLabel>
-          <RankLabel variant="active">Name</RankLabel>
-          <RankLabel variant="active">Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
-        <LeaderBoardContent>
-          <RankLabel>Rank</RankLabel>
-          <RankLabel>Name</RankLabel>
-          <RankLabel>Score</RankLabel>
-        </LeaderBoardContent>
+        {loading ? (
+          <>
+            <LoadingContent />
+            <LoadingContent />
+            <LoadingContent />
+            <LoadingContent />
+          </>
+        ) : hasRanking ? (
+          <div className="overflow-scroll max-h-40 pr-1">
+            <div className="flex flex-col gap-2 w-full">
+              {ranking?.map(([pubKeyIndex, score], index) => {
+                const rank = index + 1;
```

Comment:
I think we want to have some mechanism where teams can be tied for a place if they have the same score, ex: https://github.com/jubmoji/nfctap.xyz/blob/e9dddff7d525b1adf76a73b34ce5e486bf08247c/components/screens/HomeScreen.tsx#L119

Reply:
@AndrewCLu makes sense! i open a issue for this one